docs/src/pages/components/slider/ContinuousSlider.js

```diff
@@ -20,30 +21,56 @@ export default function ContinuousSlider() {
     setValue(newValue);
   };
 
+  const theme = {
```

Comment:
These changes are added just for testing, will be reverted before merging

Reply:
A reminder to remove the change from the demo. Maybe we could transform that into tests?

Reply:
Now that the components are moved to the lab these are separate examples, but it makes sense to move these in the tests now :))

Reply:
Added tests, not sure if it makes sense to keep that per component, but at least we have it for now to make sure we don't regress during development